services/app/src/components/roleSelection/RoleSelectionContent.jsx

```diff
@@ -1,26 +1,53 @@
-import React from 'react';
+import React, { useRef, useEffect } from 'react';
 import './role-selection.css';
 import { PlayerRoles } from '../../constants/playerConstants';
 import RoleSelectionCard from './RoleSelectionCard';
 
-const RoleSelectionContent = () => (
-  <div className='role-selection-content border-r-4 margin-auto'>
-    <p className='role-selection-title font-size-xxl weight-800 margin-0'>Choose your role</p>
-    <p className='role-selection-description'>
-      We need to know whether you participate in the estimating or not
-    </p>
-    
-    <div className='role-selection-btn-container f-row-between align-stretch'>
-      <RoleSelectionCard 
-        playerRole={PlayerRoles.Voter}
-        roleDescription="Voters estimate tasks in story points. Choose this if you're an Engineer or similar."
-      />
-      <RoleSelectionCard 
-        playerRole={PlayerRoles.Spectator}
-        roleDescription="Spectators don't participate in task estimating. Choose this if you're in Management or similar."
-      />
+const RoleSelectionContent = () => {
+  const spectatorButtonRef = useRef();
+  const voterButtonRef = useRef();
+
+  useEffect(() => {
+    voterButtonRef.current.focus();
+  }, []);
+
+  const onVoterFocusChange = (e) => {
+    if (e.key === 'Tab' || e.keyCode === '9') {
+      e.preventDefault();
+      spectatorButtonRef.current.focus();
+    }
+  }
+
+  const onSpectatorFocusChange = (e) => {
+    if (e.key === 'Tab') {
```

Comment:
why checking Tab is needed in the first place? You called method `onTabPress` on Role selection card, which means that callback should only be called when Tab button is pressed